Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = [
 ]
 
 [workspace.package]
-version = "1.1.0"
+version = "1.1.1"
```

Comment:
you should update the package.json in the prelude project as well since we need to republish.

Reply:
good catch! thx, done